lib/logatron/kv_transformer.rb

```diff
@@ -0,0 +1,21 @@
+module Logatron
+  class KvTransformer
+    def initialize(msg)
+      @msg = msg
+    end
+
+    def transform
+      a = Array.new
+      @msg.each do |k, v|
+        if v.nil?
+          a << "#{k}: -"
+        elsif v.is_a?(String) and k =~ /^body$/
+          a << "#{k}: '#{v.gsub('\'', '"')}'"
+        else
+          a << "#{k}: #{v}"
+        end
+      end
+      a.join(', ')
```

Comment:
Replace gsub with tr.  tr is many times faster than gsub for single character replacements like this.